frontend/src/components/general/scss/fonts.scss

```diff
@@ -0,0 +1,3 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto+Slab|Open+Sans|Poppins:400italic,700italic,300,400,700);
```

Comment:
i dont understand why you would do this and add the fonts within

Reply:
wym? its just one of 1 trillion ways of importing google fonts. can you put a snippet of your preferred way?

Reply:
you do that but also add the files of fonts like `frontend/public/assets/fonts/Poppins/Poppins-BlackItalic.ttf`. i dont understand why both

Reply:
oh oops, thanks. I tried to get local assets working then did the @import instead. ill delete the assets